tests/FakeItEasy.Specs/ConfiguringPropertySetterSpecs.cs

```diff
@@ -33,9 +32,11 @@ public interface IHaveInterestingProperties
             "When assignment of a property is configured using a null expression"
                 .x(() => exception = Record.Exception(() => A.CallToSet<int>(null)));
 
-            "Then the call to configure throws"
-                .x(() => exception.Should().BeAnExceptionOfType<ArgumentNullException>()
-                    .And.ParamName.Should().Be("propertySpecification"));
+            "Then an argument null exception is thrown"
+                .x(() => exception.Should().BeAnExceptionOfType<ArgumentNullException>());
+
+            "And the parameter name should be 'propertySpecification'"

```

Comment:
Yep, agreed. Good spot. I'll amend a bit later
On 14 May 2016 12:46, "Thomas Levesque" <notifications@github.com> wrote:

> In tests/FakeItEasy.Specs/ConfiguringPropertySetterSpecs.cs
> <https://github.com/FakeItEasy/FakeItEasy/pull/704#discussion_r63275041>:
>
> > @@ -33,9 +32,11 @@ public interface IHaveInterestingProperties
> >              "When assignment of a property is configured using a null expression"
> >                  .x(() => exception = Record.Exception(() => A.CallToSet<int>(null)));
> >
> > -            "Then the call to configure throws"
> > -                .x(() => exception.Should().BeAnExceptionOfType<ArgumentNullException>()
> > -                    .And.ParamName.Should().Be("propertySpecification"));
> > +            "Then an argument null exception is thrown"
> > +                .x(() => exception.Should().BeAnExceptionOfType<ArgumentNullException>());
> > +
> > +            "And the parameter name should be 'propertySpecification'"
>
> Didn't we decide earlier that step descriptions shouldn't contain "should"
> ? "And the parameter name is 'propertySpecification'"
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FakeItEasy/FakeItEasy/pull/704/files/80022e82dab8d9ccd9fedc96db1ce60801872b75#r63275041>
>